src/blockchain/log-processors/market-created.ts

```diff
@@ -30,10 +32,10 @@ export function processMarketCreatedLog(db: Knex, augur: Augur, trx: Knex.Transa
           marketCreator:              log.marketCreator,
           creationBlockNumber:        log.blockNumber,
           creationFee:                log.marketCreationFee,
-          creationTime:               blocksRows![0]!.blockTimestamp,
+          creationTime:               timestamp,
           marketType:                 extraInfo!.marketType,
-          minPrice:                   extraInfo!.minPrice,
-          maxPrice:                   extraInfo!.maxPrice,
+          minPrice:                   new BigNumber(extraInfo!.minPrice, 10).toFixed(),
```

Comment:
This converts a base-10 string to a base-10 string, is it to cut leading zeroes or to make sure that there is no decimal?

Reply:
Oh, good point -- the extraInfo fields aren't base-16.  There shouldn't be a need for this conversion at all then, I will remove.